extensions/blocks/videopress/editor.js

```diff
@@ -86,43 +86,10 @@ const addVideoPressSupport = ( settings, name ) => {
 			...settings,
 
 			attributes: {
-				autoplay: {
-					type: 'boolean',
-				},
-				caption: {
-					type: 'string',
-					source: 'html',
-					selector: 'figcaption',
-				},
-				controls: {
-					type: 'boolean',
-					default: true,
-				},
+				...settings.attributes,
```

Comment:
Could do

```suggestion
				...attributes,
```

since `attributes` is being destructured on [R66](https://github.com/Automattic/jetpack/pull/16627/files#diff-42db4cd7f3cf5ac90a135c78c26f74e5R66).